components/logiVehicle/fn_createLogiSpawnAction.sqf

```diff
@@ -52,13 +52,13 @@ private _modifierCode =
 	}
 	else
 	{
-		if (_gear isEqualTo "") then
+		if ((_gearOrCode isEqualTo "") or (typeName _gearOrCode isEqualTo "CODE")) then
```

Comment:
To compare variable types, use [`isEqualType`](https://community.bistudio.com/wiki/isEqualType) instead of `typeName _gearOrCode isEqualTo`.

This is especially important when using `isEqualTo` to compare strings (instead of `==`), as `isEqualTo` checks case-sensitive (you probably don't want that).

Reply:
Thank you, done now.

WRT the other point - implicit case-insensitive via `==` is a bit arcane so I've preferred to normalise case explicitly before comparison wherever that's been needed in CAFE.

I've found a bunch of other instances of this "typeName isEqualTo" pattern throughout the framework so I'll put a ticket in for it.